src/foundation.cs

```diff
@@ -8058,6 +8058,14 @@ interface NSString2 : NSSecureCoding, NSMutableCopying, CKRecordValue
 		[Export ("paragraphRangeForRange:")]
 		NSRange GetParagraphRange (NSRange range);
 
+		[Watch (2,0), TV (9,0), Mac (10,0), iOS (2,0)]
```

Comment:
the versions inside the attributes are all below the minimum we support, i.e. it's not needed (as it won't help the IDE experience)